hangman.js

```diff
@@ -1 +1,21 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("Welcome to Robizzy's HangMan console game.");
+
+var word = prompt("Please enter a word");
+var attempt = [];
+var incorrectAttemptLimit = 7;
+var alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h',
```

Comment:
Instead of checking a user's guess against the alphabet, what if we checked it against the user's word split into an array?